dat/missions/shadow/shadowvigil.lua

```diff
@@ -262,7 +262,11 @@ end
 -- Function hooked to jumpin. Handles most of the events in the various systems.
 function jumpin()
     sysclear = false -- We've just jumped in, so the ambushers, if any, are not dead.
-    
+
+    if pahook then -- Remove the hook on the player being attacked, if needed
+        hook.rm( pahook )
+    end    
+
```

Comment:
FYI, as of https://github.com/naev/naev/commit/caf4e168f2b14a72eb2c70f31cdc2f5ee5300dd0 it's safe to call hook.rm(pahook) without testing for nil.
Checking anyway isn't an issue, of course.